tests/integration/components/build-messages-test.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('build-messages', 'Integration | Component | build messages', {
+  integration: true
+});
+
+test('it shows the number of build messages', function(assert) {
+  assert.expect(2);
+
+  this.set('buildErrors', []);
+  this.set('isBuilding', false);
+  this.set('notify', {
+    info() {}
+  });
+
+  this.render(hbs`{{build-messages buildErrors=buildErrors isBuilding=isBuilding notify=notify}}`);
+
+  assert.equal(this.$('span').text().replace(/\s+/g, " ").trim(), 'Output ( build ok. )', 'shows build ok when no buildErrors');
+
+  this.set('buildErrors', ['error1', 'error2']);
+
+  assert.equal(this.$('span').text().replace(/\s+/g, " ").trim(), 'Output ( 2 build errors )', 'shows number of build errors');
+});
+
+test('it calls notify.info() when clicking on build errors', function(assert) {
+  assert.expect(1);
+
+  let notifyObject = Ember.Object.create({
+    called: false,
+    info() {
+      this.set('called', true);
+    }
+  });
+
+  this.cacheConsole = console.error;
```

Comment:
This should be in the `beforeEach` so that when we cleanup in the `afterEach` we will always have it available.